src/lib/common/Kconfig

```diff
@@ -32,6 +32,11 @@ config DETECT_BOARD_NAME
 config PLATFORM_LINUX
 	bool
 
+config PLATFORM_UNIX
+        bool "unix"
+	depends on HAVE_UNIX
```

Comment:
Damn. It didn't show because of 8-space tabs. My OS X's Emacs isn't properly configured for space mode.